velox/docs/configs.rst

```diff
@@ -133,6 +133,11 @@ Expression Evaluation Configuration
      - false
      - Whether to track CPU usage for individual expressions (supported by call and cast expressions). Can be expensive
        when processing small batches, e.g. < 10K rows.
+   * - legacy_cast
+     - bool
+     - false
+     - This flag applies legacy cast semantics if set to true. CAST(timestamp AS varchar) does not use space but character
```

Comment:
drop "This flag", start with a verb

Enables legacy CAST semantics if set. CAST(timestamp AS varchar) uses 'T' as a separate between date and time (instead of a space).